raiden-ts/tests/e2e/raiden.spec.ts

```diff
@@ -172,11 +172,13 @@ describe('Raiden', () => {
     test('tx fail', async () => {
       expect.assertions(1);
       // settleTimeout < min of 500
-      await expect(raiden.openChannel(token, partner, 499)).rejects.toThrow();
```

Comment:
`openChannel` is expected to be able to receive a custom `settleTimeout` per-call, besides the default one

Reply:
Wasn't aware of this requirement, will fix